src/logistics/menu.ts

```diff
@@ -95,6 +106,148 @@ export const internalCargoMenu = groupMenu({
         },
       })
     }
+
+    // Light Units
+    const unitsLightMenu = await addGroupCommandSubMenu({
+      groupName,
+      name: 'Light Units',
+      path: rootPath,
```

Comment:
Instead of putting these all under the root path, can we nest these under a `Units` menu?

so `rootPath: unitsMenu.path`

Reply:
et.all to the units sub-menus